dev/NavigationView/NavigationView.xaml

```diff
@@ -158,6 +164,19 @@
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="PaneSeperatorStates">
```

Comment:
@beervoley can you also start an email thread to review the new VisualStateGroup and state names?